.github/workflows/integration-tests.yml

```diff
@@ -9,29 +9,23 @@ name: Integation tests
       - master
 jobs:
   maven-it:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      matrix:
-        os: [ubuntu-latest]
-        java: [11, 13]
+    runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - name: Download JDK
+        run: |
+          wget --no-verbose --directory-prefix ${{ runner.temp }} \
+            https://download.oracle.com/java/17/latest/jdk-17_linux-x64_bin.tar.gz
       - name: Set up JDK
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v2
         with:
-          java-version: ${{ matrix.java }}
+          distribution: jdkfile
+          jdkFile: ${{ runner.temp }}/jdk-17_linux-x64_bin.tar.gz
+          java-version: 17
       - uses: actions/cache@v1
         with:
           path: ~/.m2/repository
-          key: ${{ runner.os }}-jdk-${{ matrix.java }}-maven-${{ hashFiles('**/pom.xml') }}
+          key: ubuntu-latest-jdk-17-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: |
-            ${{ runner.os }}-jdk-${{ matrix.java }}-maven-
+            ubuntu-latest-jdk-17-maven-
       - run: mvn -B verify -Pitcase
-      # @todo #851:30min Fix integration tests. Integration tests should start Artipie docker
```

Comment:
Why this todo is removed? Not all IT are yet rewritten with usage of Artipie docker image, corresponding ticket will be closed.

Reply:
@olenagerasimova this todo is outdated, it says that we need to enable `mvn verify` command, this is why I removed it. Now I added a new todo with a list of tests that should be fixed, see: https://github.com/artipie/artipie/pull/944/commits/75ed0792fbf3b7f41baeea61f2254e5587f17e6e